packages/sdk/src/controllers/ConnectorController.ts

```diff
@@ -2,10 +2,13 @@ import { ConnectorOptions, EditorAPI, EditorResponse, Id } from '../types/Common
 import {
     ConnectorState,
     ConnectorStateType,
-    ConnectorMapping,
+    ConnectorMappingType,

```

Comment:
why renaming this? this is a breaking change...

Reply:
It's not renaming. I've added a new type that uses as return value of ConnectorController method. The previous class with name `ConnectorMapping` still exists

Reply:
I see, my bad. It's weird UI from github :)